README.md

```diff
@@ -1,5 +1,15 @@
 ## GraphStorm
-The GraphStorm is a graph machine learning (GML) framework for enterprise use cases. It simplifies the development, training and deployment of GML models for industry-scale graphs. It provides scalable training and inference pipelines for various GML models and is optimized for extremely large graphs (measured in billons of nodes and edges). To help develop SOTA models, the framework allows very large degree of customization by providing a large collection of configurations for customizing model implementations and training pipelines as well as modularization in the training pipeline that allows users to replace every component in the pipeline.
+The GraphStorm is a graph machine learning (GML) framework for enterprise use cases.
+It simplifies the development, training and deployment of GML models for industry-scale graphs.
+It provides scalable graph loading, training and inference pipelines for various GML models
+and is optimized for extremely large graphs (measured in billons of nodes and edges).
+GraphStorm provides a collection of built-in GML models and users can train a GML model
+with a single command line without writing any code. To help develop SOTA models,
+GraphStorm provides a large collection of arguments to configure model implementations
```

Comment:
arguments or modules here?

Reply:
I think it's arguments. what do you mean by module?

Reply:
Usually we relate "arguments" to a list of arguments (values) to be passed to a script or function. Are we saying that similar to say the `nn` module of Pytorch allows developers to easily create NNs, GSF provides the building blocks for GNNs?
Or are we talking about actual arguments passed to a script/function?

Reply:
i'm talking about the actual arguments passed to a script.